scala-3.7.yaml

```
@@ -1,25 +1,28 @@
 package:
-  name: dotty
+  name: scala-3.7
   version: "3.7.1"
-  epoch: 0
+  epoch: 1
   description: The Scala 3 compiler, also known as Dotty.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - bash
     provides:
+      - dotty=${{package.full-version}}
       - scala=${{package.full-version}}
 
+vars:
+  runtime-java-version: "24"
+
 environment:
   contents:
     packages:
-      - build-base
       - busybox
-      - ca-certificates-bundle
-      - openjdk-11-default-jdk
+      - openjdk-8-default-jdk
```

Let's use a var here too for consistency, `java-version: '8'`

```suggestion
      - openjdk-${{vars.java-version}}-default-jdk
```